packages/flame_sprite_fusion/.gitignore

```diff
@@ -0,0 +1,29 @@
+# Miscellaneous
```

Comment:
Everything should be included in the root .gitignore

Reply:
Removed it.